tests/test_parsing.py

```diff
@@ -113,8 +113,59 @@ def test_get_correct_fps2():
+
+    sample = dedent(
+        r"""
+        ffmpeg version 4.2.2 Copyright (c) 2000-2019 the FFmpeg developers
+          built with Apple clang version 11.0.0 (clang-1100.0.33.8)
+          configuration: --enable-gpl --enable-version3 --enable-sdl2 --enable-fontconfig --enable-gnutls --enable-iconv --enable-libass --enable-libdav1d --enable-libbluray --enable-libfreetype --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libtheora --enable-libtwolame --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libzimg --enable-lzma --enable-zlib --enable-gmp --enable-libvidstab --enable-libvorbis --enable-libvo-amrwbenc --enable-libmysofa --enable-libspeex --enable-libxvid --enable-libaom --enable-appkit --enable-avfoundation --enable-coreimage --enable-audiotoolbox
+          libavutil      56. 31.100 / 56. 31.100
+          libavcodec     58. 54.100 / 58. 54.100
+          libavformat    58. 29.100 / 58. 29.100
+          libavdevice    58.  8.100 / 58.  8.100
+          libavfilter     7. 57.100 /  7. 57.100
+          libswscale      5.  5.100 /  5.  5.100
+          libswresample   3.  5.100 /  3.  5.100
+          libpostproc    55.  5.100 / 55.  5.100
+        Input #0, mov,mp4,m4a,3gp,3g2,mj2, from '/var/folders/82/6_ww__k94ms56ldtsph50pqc0000gn/T/imageio_sievk8ws':
+          Metadata:
+            major_brand     : mp42
+            minor_version   : 0
+            compatible_brands: isommp42
+            creation_time   : 2020-06-18T12:31:32.000000Z
+            com.android.version: 10
+          Duration: 00:00:01.07, start: 0.000000, bitrate: 2661 kb/s
+            Stream #0:0(eng): Video: h264 (High) (avc1 / 0x31637661), yuvj420p(pc, bt470bg/bt470bg/smpte170m), 720x480, 2636 kb/s, SAR 1:1 DAR 3:2, 30.01 fps, 120 tbr, 90k tbn, 180k tbc (default)
+            Metadata:
+              rotate          : 270
+              creation_time   : 2020-06-18T12:31:32.000000Z
+              handler_name    : VideoHandle
+            Side data:
+              displaymatrix: rotation of 90.00 degrees
+        Stream mapping:
+          Stream #0:0 -> #0:0 (h264 (native) -> rawvideo (native))
+        Press [q] to stop, [?] for help
+        [swscaler @ 0x7f95cd92da00] deprecated pixel format used, make sure you did set range correctly
+        Output #0, image2pipe, to 'pipe:':
+          Metadata:
+            major_brand     : mp42
+            minor_version   : 0
+            compatible_brands: isommp42
+            com.android.version: 10
+            encoder         : Lavf58.29.100
+            Stream #0:0(eng): Video: rawvideo (RGB[24] / 0x18424752), rgb24, 480x720 [SAR 1:1 DAR 2:3], q=2-31, 995328 kb/s, 120 fps, 120 tbn, 120 tbc (default)
+        """
+    )
+
+    info = parse_ffmpeg_header(sample)
+    assert info["rotate"] == 270
```

Comment:
`== "270"` ;)

Reply:
No wait. We should probably cast it to a float (in a try-except) in `_parsing.py`.

Reply:
@miguel-lorenzo any chance you could cast to a float or int in `_parsing.py`? Then the tests should pass :)

Reply:
Done!